lib/berbix.js

```diff
@@ -160,34 +167,15 @@ Client.prototype._fetchTokens = async function (path, payload) {
 };
 
 Client.prototype.createTransaction = function (opts) {
-  var payload = {};
-  if (opts.email != null) {
-    payload.email = opts.email;
-  }
-  if (opts.phone != null) {
-    payload.phone = opts.phone;
-  }
-  if (opts.customerUid != null) {
-    payload.customer_uid = "" + opts.customerUid;
-  }
-  if (opts.templateKey != null) {
-    payload.template_key = opts.templateKey;
-  }
-  if (opts.hostedOptions != null) {
-    var hostedOptions = {};
-    if (opts.hostedOptions.completionEmail != null) {
-      hostedOptions.completion_email = opts.hostedOptions.completionEmail;
-    }
-    if (opts.hostedOptions.redirectUrl != null) {
-      hostedOptions.redirect_url = opts.hostedOptions.redirectUrl;
-    }
-    payload.hosted_options = hostedOptions;
-  }
+  var payload = optionsToPayloadObject(opts)
   return this._fetchTokens("/v0/transactions", payload);
 };
 
-// createUser is deprecated, please use createTransaction
-Client.prototype.createUser = Client.prototype.createTransaction;
+Client.prototype.createHostedTransaction = async function(opts) {
+  var payload = optionsToPayloadObject(opts)
+  const tokens = await this._fetchTokens("/v0/transactions", payload)
+  return {tokens, hosted_url: tokens.response['hosted_url']}
```

Comment:
I feel like I could/should create a class for this response instead of custom building the object here.